src/components/DataKeyPair.tsx

```diff
@@ -82,6 +84,43 @@ export const DataKeyPair: FC<DataKeyPairProps> = (props) => {
   const enableClipboard = useJsonViewerStore(store => store.enableClipboard)
   const { copy, copied } = useClipboard()
 
+  const enableHighlight = useJsonViewerStore(store => store.enableHighlight)
+  const isHighlight = useMemo(() => {
+    if (!enableHighlight || prevValue === undefined) return false
+    if (typeof value !== typeof prevValue) {
+      return true
+    }
+    if (!Array.isArray(value)) {
+      if (Array.isArray(prevValue)) {
+        return true
+      }
+
+      if (typeof value === 'object' || typeof value === 'function') {
+        return false
+      }
+
+      if (value !== prevValue) {
+        return true
+      }
+    }
+    return false
+  }, [enableHighlight, prevValue, value])
+  const highlightContainer = useRef<HTMLElement>()
+  useEffect(() => {
+    if (highlightContainer.current && isHighlight) {
+      highlightContainer.current.animate(
```

Comment:
The compatability of `animate` API looks ok.
But as the fact that we need to support down to `ES5`, maybe we can do a `?.animate` or `'animatet' in highlighContainer.current`?